dev/com.ibm.ws.kernel.boot.core/src/com/ibm/ws/kernel/boot/EmbeddedServerImpl.java

```diff
@@ -380,12 +381,32 @@ public Result call() {
      * of trying to stop the server.
      */
     private class StopOperation implements Callable<Result> {
+
+        private boolean isForceShutdown = false;
+
         /**
-         * No arguments yet: someday there may be, we're just being flexible.
+         * Handles argument parsing for stop command.
          *
          * @param arguments
          */
-        StopOperation(String... arguments) {}
+        StopOperation(String... arguments) {
+            List<String> cmdArgs = new ArrayList<String>(Arrays.asList(arguments));
+
+            if (0 < cmdArgs.size()) {
+                Iterator<String> i = cmdArgs.listIterator();
+                while (i.hasNext()) {
+                    String arg = i.next();
+
+                    if (arg.equalsIgnoreCase("--force")) {
```

Comment:
I was not sure how `Arrays.asList()` worked and so I was playing with it.  I noticed that I could add null strings to the list.  I think we need to check `if ((arg != null) && (arg.equalsIgnoreCase("--force"))) {` here instead, to make sure that `arg` is not null.